### PR TITLE
[REV 29]Add localstack support

### PR DESCRIPTION
**Pull request by andrevieira11**
Ticket - https://linear.app/revent/issue/REV-29/improve-image-handling-add-support-for-localstack

### CODE REVIEW

File: commons/aws_s3.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+from django.conf import settings
+
+
+class awsS3:
+    def create_aws_client(self):
+        return boto3.resource("s3", endpoint_url=settings.AWS_S3_HOST)
+
+    # For binary files
+    def upload_file_obj(self, file, mykey):
+        s3 = self.create_aws_client()
```
Comment (g-esteves-rr) [inline commons/aws_s3.py:13]: Still need to update this class in order to reduce repeated lines of code (I think is with def  __init__(self) )

File: photo/migrations/0004_alter_contest_updated_by_alter_submission_content.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import photo.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("photo", "0003_comment_created_at_comment_updated_at_and_more"),
+    ]
+
+    operations = [
```
Comment (g-esteves-rr) [inline photo/migrations/0004_alter_contest_updated_by_alter_submission_content.py:13]: I need to make a migrations when i'm in this branch, I think this migration file should be updated with all migrations needed at the moment.

**Review by g-esteves-rr — APPROVED** [review]

**Review by kenvontucky — CHANGES_REQUESTED** [review]
look at @tiagocardoso98 comment

File: photo/migrations/0004_alter_contest_updated_by_alter_submission_content.py (hunk 2 above)
Comment (andrevieira11) [reply photo/migrations/0004_alter_contest_updated_by_alter_submission_content.py:13]: added a new migration file that should reflect the latest changes

**Comment by andrevieira11** [comment]
> 

all done!

**Review by tiagocardoso98 — APPROVED** [review]
I believe that it is ready to be merged.

**Review by kenvontucky — APPROVED** [review]

**Pull request merged**